core/scripts/Voting.js

```diff
@@ -446,6 +447,11 @@ class VotingSystem {
     const failures = [];
 
     for (const request of requests) {
+      // Stop processing requests if the batch transaction limit is reached
```

Comment:
In this case, will the AVS send more txns with the rest of the commits? Or does it just stop?

Reply:
in this case it would just stop processing the remaining txns-- should we add the new txns to a new batch?

Reply:
Fixed